### ISSUE TITLE
In App Browser not opening after logout

### ISSUE THREAD

**Issue by razmans** [recognizegroup/capacitor-plugin-msauth#38]
I am building an Android application that would enable the user to log in and log out.

The first user logs in- No problem
The first user logs out- No problem
The second user tries to log in-> The in app browser is not opening. Do you know what I am doing wrong, I have added all the necessary credentials.

My Log In function:

 `  public loginUser(): Promise<LoginResponse> {
    return new Promise((resolve, reject) => {
      const options: LoginOptions = {
        ...this.configData.mobileConfig
      };

      try {
        MsAuthPlugin.login(options).then((resultLogin: LoginResponse) => {
          resolve(resultLogin);
        });
      } catch (err) {
        reject(err);
      }
    });
  }`

My log out function
`public async logoutUser(): Promise<void> {
    await MsAuthPlugin.logout({
      clientId: this.configData.adConfig.clientId,
      tenant: this.configData.mobileConfig.tenant,
      domainHint: this.configData.mobileConfig.domainHint,
      keyHash: this.configData.mobileConfig.keyHash,
    });
  }`

**Comment by dandamoodi** [comment]
I am facing the same issue too..

**Comment by razmans** [comment]
> I am facing the same issue too.. @razmans is it resolved ?

No, am not able to resolve this. I just pushed a popup to force the user to restart the app.

**Comment by tripurarisingh1994** [comment]
Hi @razmans,

I hope this message finds you well. I recently encountered an issue with the app where it functions correctly for about 24 hours after installation, but then I receive an error message stating "**Unable to fetch access token.**" Reinstalling the app again it is working, but the problem recurs again after 24 hours.

Have you experienced this issue as well? We've been dealing with this for some time now and any insights you could provide on how you might have resolved it would be greatly appreciated.

Thank you in advance for your help!
